coalib/parsing/FilterHelper.py

```diff
@@ -21,34 +21,35 @@ def is_valid_filter(cls, filter):
         return filter in cls.available_filters
 
     @classmethod
-    def apply_filter(cls, filter, args, all_bears=None):
-        if all_bears is None:
+    def apply_filter(cls, filter_name, filter_args, bears=None):
+        if bears is None:
             from coalib.settings.ConfigurationGathering import (
                 get_all_bears)
-            all_bears = get_all_bears(LogPrinter())
-        if not cls.is_valid_filter(filter):
+            bears = get_all_bears(LogPrinter())
+        if not cls.is_valid_filter(filter_name):
             raise InvalidFilterException('{!r} is an invalid filter. '
                                          'Available filters: {}'.format(
-                                             filter, cls.get_all_filters_str()))
-        if not args or len(args) == 0:
-            return all_bears
-        return cls.available_filters[filter](all_bears, args)
+                                             filter_name,
+                                             cls.get_all_filters_str()))
+        if not filter_args or len(filter_args) == 0:
+            return bears
+        return cls.available_filters[filter_name](bears, filter_args)
 
     @classmethod
-    def apply_filters(cls, args, bears=None):
+    def apply_filters(cls, filters, bears=None):
         """
         Returns bears after filtering based on ``args``. It returns
         intersection of bears if more than one element is present in ``args``
         list.
 
-        :param bears: The bears to filter.
-        :param args:  List of args based on ``bears`` has to be filtered.
+        :param filters:  List of args based on ``bears`` has to be filtered.
                       For example:
                       ``[['language', 'c', 'java'], ['can_fix', 'syntax']]``
+        :param bears: The bears to filter.
```

Comment:
The move of this `:param bears:` entry should go into the param-reordering commit :+1: